### PR TITLE
Make the database persistent, work offline

### PR DESCRIPTION
**Pull request by Sosol02**

### CODE REVIEW

**Comment by Sosol02** [comment]
COMMENT ÇA MARCHE?
Dans le build.gradle y a une variable USE_PROD_DB. Vous la mettez à "true" au lieu de "false" et ça ira se connecter sur la db de prod.
ET APRÈS IL FAUT SYNC GRADLE POUR QUE ÇA SOIT PRIS EN COMPTE

**Review by hugolan — APPROVED** [review]
good code, nice functionality implementation

**Review by Ef55 — APPROVED** [review]

File: app/src/androidTest/java/com/github/onedirection/navigation/FavoriteAndSwipeEventTest.java [hunk 1]
```diff
@@ -68,6 +69,7 @@
     @Rule
     public ActivityTestRule<NavigationActivity> mActivityTestRule = new ActivityTestRule<>(NavigationActivity.class);
 
+    @Ignore("Jsp ce test marche pas alors que j'ai rien changé faut demander à Hugo de prier à dieu")
```
Comment (justagoodguy-omega) [inline app/src/androidTest/java/com/github/onedirection/navigation/FavoriteAndSwipeEventTest.java:72]: If you are talking about the divinity of one of the three major monotheistic religions I believe the d should be in capital letters

**Review by justagoodguy-omega — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/com/github/onedirection/navigation/FavoriteAndSwipeEventTest.java (hunk 1 above)
Comment (Sosol02) [reply app/src/androidTest/java/com/github/onedirection/navigation/FavoriteAndSwipeEventTest.java:72]: SDP's existence is proof that god doesn't exist.

**Pull request merged**